packages/abap-deploy-config-inquirer/src/index.ts

```diff
@@ -35,6 +36,7 @@ async function getPrompts(
     LoggerHelper.logger = logger ?? new ToolsLogger({ logPrefix: '@sap-ux/abap-deploy-config-inquirer' });
     PromptState.isYUI = isYUI;
     PromptState.resetAbapDeployConfig();
+    AbapServiceProviderManager.resetIsDefaultProviderAbapCloud();
```

Comment:
What does this mean? `AbapServiceProviderManager.resetIsDefaultProviderAbapCloud`